src/current/v23.2/log-sql-statistics-to-datadog.md

```diff
@@ -82,12 +89,33 @@ SET CLUSTER SETTING sql.telemetry.query_sampling.max_event_frequency = 100000;
 The `sql.telemetry.query_sampling.max_event_frequency` cluster setting and the `buffering` options in the `logs.yaml` control how many events are emitted to Datadog and that can be potentially dropped. Adjust this setting and these options according to your workload, depending on the size of events and the queries per second (QPS) observed through monitoring.
 {{site.data.alerts.end}}
 
-## Step 4. Monitor TELEMETRY logs in Datadog
+## Step 4. Configure CockroachDB to emit query and transaction events (optional)
+
+Enable the [`sql.telemetry.query_sampling.enabled` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-query-sampling-enabled) so that executed queries and transactions will emit an event on the telemetry [logging channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.telemetry.query_sampling.enabled = true;
+~~~
+
+Set the [`sql.telemetry.query_sampling.mode` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to `transaction` so that `sampled_query` and `sampled_transaction` events are emitted:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.telemetry.query_sampling.mode = 'transaction';
+~~~
+
+Configure the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) to values that are dependent on the level of granularity you require and how much performance impact from frequent logging you can tolerate:
+
+- `sql.telemetry.transaction_sampling.max_event_frequency` (default `8`) is the max event frequency (events per second) at which we sample transactions for telemetry. If sampling mode is set to 'statement', this setting is ignored. In practice, this means that we only sample a transaction if 1/max_event_frequency seconds have elapsed since the last transaction was sampled. 
+- `sql.telemetry.transaction_sampling.statement_events_per_transaction.max` (default `50`) is the maximum number of statement events to log for every sampled transaction. Note that statements that are logged by force do not adhere to this limit.
```

Comment:
this is another non-public setting

this setting should really be made public if we're going to mention it in the docs

recommend following up with engineering to get these settings flipped to public

it's really confusing that we are asking the user to employ a mix of public and non-public settings to accomplish this task

hopefully these settings get flipped to public soon

Reply:
This setting is public now. I added anchor link.